crates/apollo-compiler/test_data/diagnostics/0050_directives_in_invalid_locations.txt

```diff
@@ -2,7 +2,7 @@
     UnsupportedLocation(
         UnsupportedLocation {
             ty: "skip",
-            dir_loc: "QUERY",
+            dir_loc: Query,
```

Comment:
this changed only because it uses the Debug impl. The actual error messages displayed to the user are still all-caps-with-underscores